mir_eval/melody.py

```diff
@@ -202,7 +224,7 @@ def resample_melody_series(times, frequencies, voicing,
     frequencies : np.ndarray
         Array of frequency values, >= 0
     voicing : np.ndarray
-        Boolean array which indicates voiced or unvoiced
+        Array which indicates voiced or unvoiced
```

Comment:
Would it be worth mentioning that the voicing array does not need to be binary here?